Tests/Recurse/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 add_executable(Recurse Recurse.c)
+set_target_properties(FunctionCall PROPERTIES LINK_FLAGS "-Wl,--plugin-opt=emit-llvm")
+target_compile_options(FunctionCall PRIVATE "-flto")
```

Comment:
Needs to go before merging

Reply:
Fixed all these by changing root cmake list function command "set_target_properties" to "LINK_FLAGS" instead of "LINKER_OPTIONS"